index.html

```diff
@@ -35,12 +35,235 @@ <h2 class="headline-title2">I'm a Software Developer!</h2>
+            </article>
+        </section>
+
+        <section class="center-content">
+            <article class="content-desc">
+                <h2>Profesional Art <br />Printing Data</h2>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. Has been the industry's standard.
+                </p>
+                <ul>
+                    <li>
+                        <a href="#">HTML</a>
+                    </li>
+                    <li>
+                        <a href="#">BOOTSTRAP</a>
+                    </li>
+                    <li>
+                        <a href="#">RUBY</a>
+                    </li>
+                </ul>
+                <button type="button">See project</button>
+            </article>
+        </section>
+
+        <section class="center-content">
+            <article class="content-desc">
+                <h2>Profesional Art <br />Printing Data</h2>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. Has been the industry's standard.
+                </p>
+                <ul>
+                    <li>
+                        <a href="#">HTML</a>
+                    </li>
+                    <li>
+                        <a href="#">BOOTSTRAP</a>
+                    </li>
+                    <li>
+                        <a href="#">RUBY</a>
+                    </li>
+                </ul>
+                <button type="button">See project</button>
+            </article>
+        </section>
+
+        <section class="center-content">
+            <article class="content-desc">
+                <h2>Profesional Art <br />Printing Data</h2>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. Has been the industry's standard.
+                </p>
+                <ul>
+                    <li>
+                        <a href="#">HTML</a>
+                    </li>
+                    <li>
+                        <a href="#">BOOTSTRAP</a>
+                    </li>
+                    <li>
+                        <a href="#">RUBY</a>
+                    </li>
+                </ul>
+                <button type="button">See project</button>
+            </article>
+        </section>
+
+        <section class="about">
+            <article class="about-content">
+                <div class="about-header">
+                    <h2>About Me</h2>
+                    <p>
+                        Hello I’m a software developer! I can help you build a product,
+                        feature or website Look through some of my work and experience! If
+                        you like what you see and have a project you need coded, don’t
+                        hesitate to contact me.
+                    </p>
```

Comment:
- [ ] As I mentioned above :point_up_2: please stick to the Figma design your paragraph is not the same as the Figma design I think you use `text-align:  justify` in your `CSS` please remove it if you use :+1: check below point_down the comparison between your design and Figma design :slightly_smiling_face:

| Your Design | Figma Design |
|--------|--------|
| ![Screenshot from 2023-04-30 01-34-58](https://user-images.githubusercontent.com/45226848/235324105-ddbd9610-43c4-430e-975a-c9ec52af875a.png) | ![Screenshot from 2023-04-30 01-34-48](https://user-images.githubusercontent.com/45226848/235324112-4ee3bcfd-ee20-48b4-965d-1f17cc5a24b1.png) |